packages/core-entities/src/Definitions.ts

```diff
@@ -269,7 +269,11 @@ export interface YieldData {
   feeAPY?: number;
   strategyAPY?: number;
   incentives?: {
-    tokenId: string;
```

Comment:
@matthew-garrett Can we just rename this `incentives` prop to be `noteIncentives` We're assuming that in the code below anyway.